src/components/SkipButton.js

```diff
@@ -0,0 +1,13 @@
+function SkipButton({ skipTo }) {
+    return(
+        <>
+            <div className="skip-proj-btn-wrapper">
+                <a href={`#${skipTo}`} className="skip-proj-btn secondary-btn">
+                Skip Projects
+                </a>     
+            </div>          
+        </>
+    )
+}
+
+export default SkipButton;
```

Comment:
Can you please rename this component/file "SkipSectionButton", that way it's more specific and can possibly be reused for any sections that need a skip button(e.g. navigation)